src/NodeApi/JSValue.cs

```diff
@@ -194,11 +193,15 @@ public static unsafe JSValue SymbolFor(ReadOnlySpan<byte> utf8Name)
         func.AddGCHandleFinalizer((nint)descriptorHandle);
         return func;
     }
-#endif
 
     public static unsafe JSValue CreateFunction(
-        string name, JSCallback callback, object? callbackData = null)
+        string? name, JSCallback callback, object? callbackData = null)
     {
+        if (name is null)
+        {
+            return CreateFunction((byte*)null, 0, callback, callbackData);
+        }
+
 #if NETFRAMEWORK
         int byteCount = Encoding.UTF8.GetByteCount(name);
         byte* utf8Name = stackalloc byte[byteCount];
```

Comment:
>stackalloc [](http://example.com/codeflow?start=25&length=10)

Not related to this PR. I just noticed it. are we allowed to use stackalloc for the .Net Framework 4.x? Is it just a C# feature instead of the framework?